src/Forms/DateField.php

```diff
@@ -96,14 +106,6 @@ class DateField extends TextField
      */
     protected $maxDate = null;
 
-    /**
-     * Unparsed value, used exclusively for comparing with internal value
-     * to detect invalid values.
-     *
-     * @var mixed
-     */
-    protected $rawValue = null;
```

Comment:
While I am glad to see we're not storing two sets of the value now... please explain what this was used for and why we don't need it anymore?

Reply:
On DateField/DatetimeField/TimeField it was was an entirely unmodified value that came in from setValue() / setSubmittedValue(). Then the internal value $this->value was changed due to all the localisation fun that happens.  rawValue was then checked to see if the field was empty all along and to not validate it if was.

tl;dr it was pointless

Reply:
Sweet, sounds like it was a non-standard workaround for the way the value property was being (ab)used. I'm okay with it being axed.